src/courtroom_widgets.cpp

```diff
@@ -1134,111 +1128,84 @@ int Courtroom::adapt_numbered_items(QVector<T *> &item_vector,
 
 void Courtroom::check_effects()
 {
+  // Asset lookup order
+  // 1. In the character folder, look for
+  // `effect_names.at(i)` + extensions in `exts` in order
+  // 2. In the theme folder (gamemode-timeofday/main/default), look for
+  // `effect_names.at(i)` + extensions in `exts` in order
+  // Only enable buttons where a file was found
+
+  QStringList exts{".webp", ".gif", ".apng"};
```

Comment:
Honestly, we could probably lift the extensions out to a header, and just refer to that, instead of defining it in every function.

Reply:
can fix